test/models/movie.test.js

```diff
@@ -19,4 +19,88 @@ describe('movie model', () => {
 
   });
 
+  describe('filterByYear', () => {
```

Comment:
Out of scope as you said, but if you wanted to clear database state before tests you could use a `beforeEach` and a `TRUNCATE`.

You'll see this a lot in our other APIs:

```js
beforeEach(() => {
  return Knex.raw('TRUNCATE movies CASCADE');
});
```